packages/components/src/__utilities__/Focusable/v3/Focusable.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { HTMLAttributes, ReactNode, useRef } from "react"
+import classnames from "classnames"
+import { FocusableOptions, useFocusable } from "react-aria"
+import styles from "./Focusable.module.scss"
+
+export type FocusableProps = {
+  children: ReactNode
+} & FocusableOptions &
+  HTMLAttributes<HTMLDivElement>
+
+// todo this component will be reusable util for all components to give them focusable wrapper
+export const Focusable = ({
+  children,
+  className,
+  ...props
+}: FocusableProps): JSX.Element => {
+  const ref = useRef<HTMLDivElement>(null)
+  const { focusableProps } = useFocusable(props, ref)
+
+  return (
+    <div
+      ref={ref}
+      className={classnames(styles.focusableWrapper, className)}
+      {...focusableProps}
+      data-inline-hidden-content
+      // We want the div to be focusable for keyboard users,
+      // but screen readers will have the VisuallyHidden content
+      // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
+      tabIndex={0}
+      // aria-describedby on div doesn't do anthing so we instead render the content in VisuallyHidden from tooltip
+      // but because RAC adds it as it assumes it's interactive element we remove it here
+      aria-describedby={undefined}
```

Comment:
Given this is a util that could be used for anything, it doesn't seem ideal that something specific to tooltip is baked in here. Are there any other options?